CONTRIBUTING.md

```diff
@@ -21,6 +21,8 @@ This repository contains the conceptual documentation for .NET. The .NET documen
 - [.NET Compiler Platform SDK reference](https://github.com/dotnet/roslyn-api-docs)
     Issues and tasks for this repo are tracked in [dotnet/docs/issues](https://github.com/dotnet/docs/issues).
 
+[!INCLUDE [](includes/loc-only-mt-engine-updates.md)]
```

Comment:
The contributing file is a repo only file that doesn't accept Markdown extensions. So this would show up for customers reading this.

Reply:
Do you mean that INCLUDES reference won't work for CONTRIBUTING.md, so I should directly add the content?

Reply:
Yep. This what customers will see right now: https://github.com/dotnet/docs/blob/3b24abde90b95f531799197b392667706b0aa7c3/CONTRIBUTING.md

Reply:
I see, thank you. I've replaced the INCLUDES reference with proposed content and deleted the INCLUDES file from this PR.